yarn.lock

```diff
@@ -1740,10 +1740,10 @@
     tslib "^2.4.0"
     undici "^5.5.1"
 
-"@elastic/transport@^8.3.2":
-  version "8.3.2"
-  resolved "https://registry.yarnpkg.com/@elastic/transport/-/transport-8.3.2.tgz#295e91f43e3a60a839f998ac3090a83ddb441592"
-  integrity sha512-ZiBYRVPj6pwYW99fueyNU4notDf7ZPs7Ix+4T1btIJsKJmeaORIItIfs+0O7KV4vV+DcvyMhkY1FXQx7kQOODw==
+"@elastic/transport@^8.3.3":
```

Comment:
__nit:__ it looks like we need to join this declaration with the declaration above as they are semver-compatible (`@elastic/transport@^8.3.1`). Yarn sometimes cannot do this automatically, so we need to manually remove declarations from `yarn.lock` and re-run `yarn kbn bootstrap`.
```
"@elastic/transport@^8.3.1", "@elastic/transport@^8.3.3":
  version "8.3.3"
```

Reply:
Good catch! Done in [`bdeaf61` (#163822)](https://github.com/elastic/kibana/pull/163822/commits/bdeaf618b524577fd46f7c369ce1eb9df5b736d9)